src/store/reducers/index.js

```diff
@@ -1,12 +1,6 @@
-const initialState = {
-    articles: [],
-    notifications: [],
-    followers: [],
-}
+import { combineReducers } from 'redux';
+import loginReducer from './LoginReducer';
 
-function rootReducer(state = initialState, action){
-
-    return state;
-};
-
-export default rootReducer;
+export default combineReducers({
```

Comment:
Can I suggest this, It is more readable:
```suggestion
const rootReducer = combineReducers({
login_user: loginReducer})
export default rootReducer
```